packages/shared/src/components/squads/PublicSquadSubmissionActions.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { ReactElement, useCallback } from 'react';
+import { Button, ButtonSize, ButtonVariant } from '../buttons/Button';
+import { squadsPublicGuide } from '../../lib/constants';
+import { anchorDefaultRel } from '../../lib/strings';
+import { SquadStatus } from './settings';
+import { TimerIcon } from '../icons';
+import { usePublicSquadRequests } from '../../hooks';
+import { useLazyModal } from '../../hooks/useLazyModal';
+import { PUBLIC_SQUAD_REQUEST_REQUIREMENT } from '../../lib/config';
+import { LazyModal } from '../modals/common/types';
+import { Squad } from '../../graphql/sources';
+
+interface PublicSquadSubmissionActionsProps {
+  squad: Squad;
+  isDetailsVisible?: boolean;
+}
+
+const PublicSquadSubmissionActions = (
+  props: PublicSquadSubmissionActionsProps,
+): ReactElement => {
+  const { squad, isDetailsVisible = true } = props;
+  const { isFetched, status } = usePublicSquadRequests({
+    sourceId: squad?.id,
+  });
+  const { openModal } = useLazyModal();
+  const postsCount = squad?.flags?.totalPosts ?? 0;
+  const isEligible = postsCount >= PUBLIC_SQUAD_REQUEST_REQUIREMENT;
```

Comment:
It's non blocking, but maybe consideration if we can export it somewhere as a shared function? (not sure if we already have a hook or something we can re-use?

Reply:
Current hooks do not accept the post count. 
- I can move this piece of code in a shared function but maybe it will be lost in the codebase and it will be hard to be found. 
- Another option is change the arguments used by `usePublicSquadRequests` and insert there, this should be ok?

Reply:
Also fine to leave it.
Just want to avoid if things change to eval criteria we only modify 1 space by accident.